components/KitButtonSupreme.js

```diff
@@ -1,54 +1,61 @@
+/* Use this button for any round buttons, either filled or outlined
+If you want text that acts as a button, do not use KitButtonSupreme.
+Use KitText wrapped by TouchableOpacity instead */
+
 import React from "react";
 import { View, StyleSheet, Image, TouchableOpacity } from "react-native";
 import Colors from "../constants/Colors";
 import KitText from "./KitText";
 
-// Button can either be 'filled' or 'outlined
-// Actual correct solution is to have a type checker + documentation but it be like that
-export default function KitButtonSupreme(props) {
-  let type = props.type || "filled";
-  if (props.type !== "filled" && props.type !== "outlined") {
-    type = "filled"; //  Use filled if anything other than filled and outlined is passed in
-  }
+export default function KitButtonSupreme({
+  onPress,
```

Comment:
I really like this way of doing props. Thanks for surfacing!

Reply:
Oh yeah, Bia saw this somewhere